DECISIONLOG.md

```diff
@@ -1 +1,2 @@
 # Decision Log 
+ 
```

Comment:
Is there a need for a decisionlog in this package?

Reply:
not yet! removed